randobot/friendly_names.py

```diff
@@ -0,0 +1,139 @@
+class FriendlyNames:
+    draftables = {
+        "bridge": {
+            "__setting": "Rainbow Bridge",
+            "open": "Open bridge, 6 med GCBK"
+        },
+        "deku": {
+            "__setting":"Kokiri Forest",
+            "open": "Open Forest",
+        },
+        "interiors": {
+            "__setting":"Indoor Entrance Randomizer",
+            "on": "Enabled",
+        },
+        "dungeons": {
+            "__setting":"Dungeon Entrance Randomizer",
+            "on": "Simple (no Ganon's Castle)",
+        },
+        "grottos": {
+            "__setting":"Grotto Entrance Randomizer",
+            "on": "Enabled",
+        },
+        "shops": {
+            "__setting":"Shopsanity",
+            "on": "4 items, random prices",
+        },
+        "ow_tokens": {
+            "__setting":"Overworld Tokens",
+            "on": "Shuffled",
+        },
+        "dungeon_tokens": {
+            "__setting":"Dungeon Tokens",
+            "on": "Shuffled",
+        },
+        "scrubs": {
+            "__setting":"Scrub shuffle",
+            "on": "Enabled, affordable prices",
+        },
+        "keys": {
+            "__setting":"Small Keys",
+            "keysy": "Keysy (dungeon small keys and boss keys removed)",
```

Comment:
I've just ported these friendly names over from the discord bot. I'm pretty sure they could be made more succinct. Alternatively probably want to limit the size of select boxes in the rtGG chat because these get way too wide.

Reply:
Resolved by companion PR in racetime-app linked below.